2020-11-02 16:56:26,644 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-02 16:56:36,177 INFO __main__ VS-withbase ball_cross_template 0 || loading data for generative training...
2020-11-02 16:56:43,490 INFO __main__ VS-withbase ball_cross_template 0 || training 'generative' models...
2020-11-02 23:19:26,720 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-02 23:19:31,398 INFO __main__ VS-withbase ball_cross_template 0 || loading data for generative training...
2020-11-02 23:19:36,841 INFO __main__ VS-withbase ball_cross_template 0 || training 'generative' models...
2020-11-02 23:19:36,852 ERROR __main__ Exception occured:
Traceback (most recent call last):
  File "solver.py", line 259, in <module>
    solver.train_supervised(epochs=epochs, train_mode=train_mode, test_ids = test_ids, setup=f'{eval_setup}_{fold_id}')
  File "/media/compute/homes/aharter/isy2020/phyre/flownet.py", line 1834, in train_supervised
    L.info(f"training epoch {epoch+1}...")
NameError: name 'L' is not defined
2020-11-02 23:22:19,029 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-02 23:22:24,925 INFO __main__ VS-withbase ball_cross_template 0 || loading data for generative training...
2020-11-02 23:22:31,498 INFO __main__ VS-withbase ball_cross_template 0 || training 'generative' models...
2020-11-02 23:22:40,484 ERROR __main__ Exception occured:
Traceback (most recent call last):
  File "solver.py", line 259, in <module>
    solver.train_supervised(epochs=epochs, train_mode=train_mode, test_ids = test_ids, setup=f'{eval_setup}_{fold_id}')
  File "/media/compute/homes/aharter/isy2020/phyre/flownet.py", line 1834, in train_supervised
    L.info(f"training epoch {epoch+1}...")
NameError: name 'L' is not defined
2020-11-02 23:24:46,511 ERROR __main__ Exception occured:
Traceback (most recent call last):
  File "solver.py", line 202, in <module>
    neck=neckfak, altconv=altconv, train_mode=train_mode)
  File "/media/compute/homes/aharter/isy2020/phyre/flownet.py", line 553, in __init__
    L = logging.getLogger(__name__)
NameError: name 'logging' is not defined
2020-11-02 23:25:12,276 ERROR __main__ Exception occured:
Traceback (most recent call last):
  File "solver.py", line 202, in <module>
    neck=neckfak, altconv=altconv, train_mode=train_mode)
  File "/media/compute/homes/aharter/isy2020/phyre/flownet.py", line 553, in __init__
    L = logging.getLogger(__name__)
NameError: name 'logging' is not defined
2020-11-02 23:25:38,646 ERROR __main__ Exception occured:
Traceback (most recent call last):
  File "solver.py", line 202, in <module>
    neck=neckfak, altconv=altconv, train_mode=train_mode)
  File "/media/compute/homes/aharter/isy2020/phyre/flownet.py", line 553, in __init__
    L = logging.getLogger(__name__)
NameError: name 'logging' is not defined
2020-11-02 23:29:13,933 ERROR __main__ Exception occured:
Traceback (most recent call last):
  File "solver.py", line 202, in <module>
    neck=neckfak, altconv=altconv, train_mode=train_mode)
  File "/media/compute/homes/aharter/isy2020/phyre/flownet.py", line 553, in __init__
    L = logging.getLogger(__name__)
NameError: name 'logging' is not defined
2020-11-02 23:29:39,554 ERROR __main__ Exception occured:
Traceback (most recent call last):
  File "solver.py", line 202, in <module>
    neck=neckfak, altconv=altconv, train_mode=train_mode)
  File "/media/compute/homes/aharter/isy2020/phyre/flownet.py", line 553, in __init__
    L = logging.getLogger(__name__)
NameError: name 'logging' is not defined
2020-11-02 23:31:27,332 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-02 23:31:31,746 INFO __main__ VS-withbase ball_cross_template 0 || loading data for generative training...
2020-11-02 23:31:36,864 INFO __main__ VS-withbase ball_cross_template 0 || training 'generative' models...
2020-11-02 23:31:38,847 INFO root training epoch 1...
2020-11-02 23:32:00,244 INFO root collecting auccess after epoch 1...
2020-11-02 23:32:05,787 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-02 23:42:13,379 INFO root finish collecting auccess after epoch 1: 0.22104924787133576
2020-11-02 23:42:13,379 INFO root training epoch 2...
2020-11-02 23:42:34,573 INFO root collecting auccess after epoch 2...
2020-11-02 23:42:39,903 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-02 23:52:09,782 INFO root finish collecting auccess after epoch 2: 0.3016581793372354
2020-11-02 23:52:09,783 INFO root training epoch 3...
2020-11-02 23:52:31,155 INFO root training epoch 4...
2020-11-02 23:52:52,722 INFO root training epoch 5...
2020-11-02 23:53:14,255 INFO root collecting auccess after epoch 5...
2020-11-02 23:53:19,499 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-03 00:02:47,933 INFO root finish collecting auccess after epoch 5: 0.2940313229862488
2020-11-03 00:02:47,934 INFO root training epoch 6...
2020-11-03 00:03:10,595 INFO root training epoch 7...
2020-11-03 00:03:32,295 INFO root training epoch 8...
2020-11-03 00:03:54,688 INFO root training epoch 9...
2020-11-03 00:04:16,580 INFO root training epoch 10...
2020-11-03 00:04:39,328 INFO root collecting auccess after epoch 10...
2020-11-03 00:04:44,412 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-03 00:14:00,995 INFO root finish collecting auccess after epoch 10: 0.36187468128104516
2020-11-03 00:14:02,890 INFO __main__ VS-withbase ball_cross_template 0 || saving models...
2020-11-03 00:14:03,155 INFO __main__ VS-withbase ball_cross_template 0 || loading data for generative testing...
2020-11-03 00:14:03,419 INFO __main__ VS-withbase ball_cross_template 0 || inspecting generative performance...
2020-11-03 00:14:03,438 ERROR __main__ Exception occured:
Traceback (most recent call last):
  File "solver.py", line 280, in <module>
    solver.inspect_supervised(eval_setup, fold_id, train_mode = train_mode)
  File "/media/compute/homes/aharter/isy2020/phyre/flownet.py", line 2448, in inspect_supervised
    action_pred = self.models["act_net"](T.cat((init_scenes, target_pred), dim=1))
  File "/media/compute/homes/aharter/miniconda3/envs/phyre/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/media/compute/homes/aharter/isy2020/phyre/flownet.py", line 472, in forward
    return self.model(X)
  File "/media/compute/homes/aharter/miniconda3/envs/phyre/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/media/compute/homes/aharter/miniconda3/envs/phyre/lib/python3.6/site-packages/torch/nn/modules/container.py", line 117, in forward
    input = module(input)
  File "/media/compute/homes/aharter/miniconda3/envs/phyre/lib/python3.6/site-packages/torch/nn/modules/module.py", line 722, in _call_impl
    result = self.forward(*input, **kwargs)
  File "/media/compute/homes/aharter/miniconda3/envs/phyre/lib/python3.6/site-packages/torch/nn/modules/conv.py", line 419, in forward
    return self._conv_forward(input, self.weight)
  File "/media/compute/homes/aharter/miniconda3/envs/phyre/lib/python3.6/site-packages/torch/nn/modules/conv.py", line 416, in _conv_forward
    self.padding, self.dilation, self.groups)
RuntimeError: Given groups=1, weight of size [8, 7, 4, 4], expected input[50, 6, 64, 64] to have 7 channels, but got 6 channels instead
2020-11-03 10:58:36,332 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-03 10:58:40,895 INFO __main__ VS-withbase ball_cross_template 0 || loading models...
2020-11-03 10:58:41,334 INFO __main__ VS-withbase ball_cross_template 0 || saving models...
2020-11-03 10:58:41,593 INFO __main__ VS-withbase ball_cross_template 0 || loading data for generative testing...
2020-11-03 10:58:41,946 INFO __main__ VS-withbase ball_cross_template 0 || inspecting generative performance...
2020-11-03 10:58:47,127 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-03 10:58:51,726 INFO __main__ VS-withbase ball_within_template 0 || loading models...
2020-11-03 10:58:51,727 ERROR __main__ Exception occured:
Traceback (most recent call last):
  File "solver.py", line 227, in <module>
    solver.load_models(setup=eval_setup, fold=fold_id, no_second_stage=no_scnd_stage)
  File "/media/compute/homes/aharter/isy2020/phyre/flownet.py", line 2742, in load_models
    self.models[model].load_state_dict(T.load(load_path+f'/{model}.pt', map_location=T.device(device)))
  File "/media/compute/homes/aharter/miniconda3/envs/phyre/lib/python3.6/site-packages/torch/serialization.py", line 571, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/media/compute/homes/aharter/miniconda3/envs/phyre/lib/python3.6/site-packages/torch/serialization.py", line 229, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/media/compute/homes/aharter/miniconda3/envs/phyre/lib/python3.6/site-packages/torch/serialization.py", line 210, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: 'saves/flownet/VS-withbase_within_0/tar_net.pt'
2020-11-03 11:19:23,708 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-03 11:19:28,315 INFO __main__ VS-withbase ball_cross_template 0 || loading data for generative training...
2020-11-03 11:19:33,694 INFO __main__ VS-withbase ball_cross_template 0 || training 'generative' models...
2020-11-03 11:19:35,786 INFO root training epoch 1...
2020-11-03 11:19:56,929 INFO root collecting auccess after epoch 1...
2020-11-03 11:20:03,295 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-03 11:24:33,884 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-03 11:24:38,616 INFO __main__ VS-withbase ball_cross_template 0 || loading data for generative training...
2020-11-03 11:24:44,180 INFO __main__ VS-withbase ball_cross_template 0 || training 'generative' models...
2020-11-03 11:24:46,412 INFO root training epoch 1...
2020-11-03 11:25:06,596 INFO root collecting auccess after epoch 1...
2020-11-03 11:25:12,369 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-03 11:35:01,758 INFO root finish collecting auccess after epoch 1: 0.3183883399781466
2020-11-03 11:35:01,759 INFO root training epoch 2...
2020-11-03 11:35:22,353 INFO root collecting auccess after epoch 2...
2020-11-03 11:35:28,453 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-03 11:45:15,707 INFO root finish collecting auccess after epoch 2: 0.3340666505730803
2020-11-03 11:45:15,707 INFO root training epoch 3...
2020-11-03 11:45:36,480 INFO root training epoch 4...
2020-11-03 11:45:57,309 INFO root training epoch 5...
2020-11-03 11:46:18,450 INFO root collecting auccess after epoch 5...
2020-11-03 11:46:24,070 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-03 11:55:57,101 INFO root finish collecting auccess after epoch 5: 0.3611160437327204
2020-11-03 11:55:57,101 INFO root training epoch 6...
2020-11-03 11:56:17,289 INFO root training epoch 7...
2020-11-03 11:56:38,598 INFO root training epoch 8...
2020-11-03 11:56:59,925 INFO root training epoch 9...
2020-11-03 11:57:21,563 INFO root training epoch 10...
2020-11-03 11:57:42,671 INFO root collecting auccess after epoch 10...
2020-11-03 11:57:48,252 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-03 12:07:10,440 INFO root finish collecting auccess after epoch 10: 0.3852969385940911
2020-11-03 12:07:11,883 INFO __main__ VS-withbase ball_cross_template 0 || saving models...
2020-11-03 12:07:12,151 INFO __main__ VS-withbase ball_cross_template 0 || loading data for generative testing...
2020-11-03 12:07:12,437 INFO __main__ VS-withbase ball_cross_template 0 || inspecting generative performance...
2020-11-03 12:07:15,551 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-03 12:07:20,322 INFO __main__ VS-withbase ball_within_template 0 || loading data for generative training...
2020-11-03 12:08:51,463 INFO __main__ VS-withbase ball_within_template 0 || training 'generative' models...
2020-11-03 12:08:51,497 INFO root training epoch 1...
2020-11-03 12:09:11,967 INFO root collecting auccess after epoch 1...
2020-11-03 12:09:17,507 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-03 12:20:17,766 INFO root finish collecting auccess after epoch 1: 0.23562032600465016
2020-11-03 12:20:17,766 INFO root training epoch 2...
2020-11-03 12:20:40,674 INFO root collecting auccess after epoch 2...
2020-11-03 12:20:46,295 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-03 12:30:08,559 INFO root finish collecting auccess after epoch 2: 0.43921494726394567
2020-11-03 12:30:08,559 INFO root training epoch 3...
2020-11-03 12:30:30,387 INFO root training epoch 4...
2020-11-03 12:30:53,394 INFO root training epoch 5...
2020-11-03 12:31:17,199 INFO root collecting auccess after epoch 5...
2020-11-03 12:31:22,820 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-03 12:39:52,942 INFO root finish collecting auccess after epoch 5: 0.5150335461896014
2020-11-03 12:39:52,943 INFO root training epoch 6...
2020-11-03 12:40:16,875 INFO root training epoch 7...
2020-11-03 12:40:40,415 INFO root training epoch 8...
2020-11-03 12:41:03,001 INFO root training epoch 9...
2020-11-03 12:41:25,945 INFO root training epoch 10...
2020-11-03 12:41:46,852 INFO root collecting auccess after epoch 10...
2020-11-03 12:41:52,553 INFO root Loading cache from /media/compute/homes/aharter/.cache/phyre/offline_simulation/100000/ball
2020-11-03 12:50:07,085 INFO root finish collecting auccess after epoch 10: 0.522334298477171
2020-11-03 12:50:08,464 INFO __main__ VS-withbase ball_within_template 0 || saving models...
2020-11-03 12:50:08,733 INFO __main__ VS-withbase ball_within_template 0 || loading data for generative testing...
2020-11-03 12:50:09,271 INFO __main__ VS-withbase ball_within_template 0 || inspecting generative performance...
2020-11-03 12:50:15,309 INFO __main__ ['ball_cross_template', 'ball_within_template']
